pom.xml

```diff
@@ -212,6 +212,7 @@
         <flink.protobuf.version>2.7.6</flink.protobuf.version>
         <flink.connector.mongodb.cdc.version>2.2.1</flink.connector.mongodb.cdc.version>
         <flink.connector.oracle.cdc.version>2.2.1</flink.connector.oracle.cdc.version>
+        <flink.connector.redis>1.1-SNAPSHOT</flink.connector.redis>
```

Comment:
no stable version of redis connetor ?

Reply:
This is a stable version, but the version name is SNAPSHOT.